packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -168,6 +168,11 @@ const chatMessageStyles: ComponentSlotStylesPrepared<ChatMessageStylesProps, Cha
       }),
     float: 'right',
   }),
+  header: () => ({
```

Comment:
Let's move this to `chatMessageHeaderStyles`

Reply:
I moved but the styles  aren't applied... What have I missed?

Reply:
You need to export them in `componentStyles`

Reply:
Thanks, done